lib/board.rb

```diff
@@ -0,0 +1,40 @@
+class Board
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9] 
+  end
+
+  # def play
+  #   puts draw_board
+  # end
+
```

Comment:
best not to commit comments for dead code without reason.  git has a history if you ever need to go back.